Changes

```diff
@@ -26,6 +26,9 @@ NEXT_RELEASE:
 - #127: install ocamlbuild's man pages, missing since 4.02
   (Gabriel Scherer)
 
+- #132: add a rule producing .cmxs from .mllib files
+  (Armaël Guéneau, requested by Daniel Bünzli)
```

Comment:
Could you add just a bit more information here for people that have not followed the discussion? Something like "(Previously `.mllib` would only produce `.cm{x,}a`, and you had to use another file `.mldylib` for `.cmxs`. Now, a `.mllib` can also produce a `.cmxs`, but `.mldylib` files still take precedence when they exist.)"